apps/asap-server/src/handlers/webhooks/webhook-calendar-created.ts

```diff
@@ -1,21 +1,18 @@
-import debug from 'debug';
-import AWS from 'aws-sdk';
+/* eslint-disable no-shadow */
```

Comment:
why do we need it exactly?

Reply:
This is a rule that makes sense when we talk about generic variables with the same name, but in different scopes. Imagine the case of a variable names `result`.
In this case, the variable we define in the outside scope, the import, and the variable inside the function scope, the parameter, are always the same.
Here, naming the variable with a different name will actually make the code harder to read, which is what the rule intends to achieve.